### ISSUE TITLE
Fatal Python error: PyGILState_Release: thread state 0x7f2fac019dd0 must be current when releasing

### ISSUE THREAD

**Issue by volgar1x** [aqora-io/cli#65]
This crash happens from time to time, and I don't have a method to reproduce it unfortunately. Here is an example of this error to happen: https://github.com/aqora-io/platform/actions/runs/9433500388/job/25984512617

**Comment by volgar1x** [comment]
It appears more easily reproducible on a single CPU VM

**Comment by volgar1x** [comment]
I was able to eliminate this bug by removing the usage of [`pyo3_asyncio/tokio/fn.run`]. This call is required in order to have a running asyncio event loop, needed especially when running `aqora test`. I was able to produce a debug binary with symbols included and it seems this function moves the GIL into a closure, delaying its drop until too late. Maybe we are not supposed to call [`pyo3_asyncio/tokio/fn.run`] inside a `#[pyfunction]`, or maybe we can fork & patch the library to make it work for us.

Anyway, this is not a critical bug when only very low-resource machines are affected (such as our platform integration runners).

[`pyo3_asyncio/tokio/fn.run`]: https://docs.rs/pyo3-asyncio/latest/pyo3_asyncio/tokio/fn.run.html
